### PR TITLE
Auto Save on Metroid Deaths

### PR DESCRIPTION
**Pull request by MattEqualsCoder**

### CODE REVIEW

**Review by Vivelin — DISMISSED** [review]

File: src/Randomizer.Data/Options/CasPatches.cs
Comment (CPColin) [inline src/Randomizer.Data/Options/CasPatches.cs:1]: "you" should be "your" unless we want to make it a Captain Novolin reference

Comment (MattEqualsCoder) [reply src/Randomizer.Data/Options/CasPatches.cs:1]: Did this somehow get attached to the wrong line or the GitHub mobile app being weird? I'm not seeing how "your" would make sense here.

Comment (CPColin) [reply src/Randomizer.Data/Options/CasPatches.cs:1]: Should be for line 77, "Save you progress". I wasn't sure it was attaching to the right line on my end, either.

Comment (MattEqualsCoder) [reply src/Randomizer.Data/Options/CasPatches.cs:1]: Ah yeah, it was showing a different line for me. lol Should be fixed now.

**Review by CPColin — APPROVED** [review]

**Pull request merged**